h2o-core/src/test/java/water/MRTaskTest.java

```diff
@@ -5,9 +5,56 @@
 import water.fvec.Chunk;
 import water.util.PrettyPrint;
 
+import java.util.concurrent.atomic.AtomicInteger;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
 public class MRTaskTest extends TestUtil {
-  @BeforeClass static public void setup() { stall_till_cloudsize(5); }
+  @BeforeClass static public void setup() { stall_till_cloudsize(1); }
```

Comment:
The reduced cloud size requirement affect also other tests in this class. Is that intentional?